File: app/src/main/java/com/boardgamegeek/ui/PlayStatsActivity.java
package com.boardgamegeek.ui;

import android.content.Intent;
import android.os.Bundle;
import android.support.v4.app.Fragment;

import com.crashlytics.android.answers.Answers;
import com.crashlytics.android.answers.ContentViewEvent;

public class PlayStatsActivity extends SimpleSinglePaneActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		if (savedInstanceState == null) {
			Answers.getInstance().logContentView(new ContentViewEvent().putContentType("PlayStats"));
		}
	}

	@Override
	protected Fragment onCreatePane(Intent intent) {
		return new PlayStatsFragment();
	}
}

File: app/src/main/java/com/boardgamegeek/ui/PlayStatsActivity.kt
package com.boardgamegeek.ui

import android.os.Bundle
import com.google.firebase.analytics.FirebaseAnalytics
import com.google.firebase.analytics.logEvent
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class PlayStatsActivity : SimpleSinglePaneActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (savedInstanceState == null) {
            firebaseAnalytics.logEvent(FirebaseAnalytics.Event.VIEW_ITEM) {
                param(FirebaseAnalytics.Param.CONTENT_TYPE, "PlayStats")
            }
        }
    }

    override fun createPane() = PlayStatsFragment()
}
